src/flamenco/vm/syscall/fd_vm_syscall_pda.c

```diff
@@ -36,9 +36,8 @@ fd_vm_derive_pda( fd_vm_t *           vm,
   fd_vm_vec_t const * seeds_haddr = FD_VM_MEM_SLICE_HADDR_LD( vm, seeds_vaddr, FD_VM_ALIGN_RUST_SLICE_U8_REF,
     fd_ulong_sat_mul( seeds_cnt, FD_VM_VEC_SIZE ) );
 
-  if ( seeds_cnt>FD_VM_PDA_SEEDS_MAX ) {
-    FD_VM_ERR_FOR_LOG_SYSCALL( vm, FD_VM_ERR_SYSCALL_BAD_SEEDS );
-    return FD_VM_ERR_INVAL;
+  if( FD_UNLIKELY( seeds_cnt+( !!bump_seed )>FD_VM_PDA_SEEDS_MAX ) ) {
```

Comment:
Should we return a distinct error here? This is a different error to invalid PDA right?